_players/kerstin-biedermann.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: player
+
+name: Kerstin Biedermann
+position: Player
+company: Robert Bosch GmbH
+bio: "Quality Key Account Manager"
+
```

Comment:
It's a good practice to always have a default avatar
`avatar: /images/players/default.png`